packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -100,7 +100,13 @@ export default class GoogleCalendarService implements Calendar {
         reminders: {
           useDefault: true,
         },
-        guestsCanSeeOtherGuests: calEventRaw.seatsShowAttendees,
+        // If there are no seats then share attendee information
+        guestsCanSeeOtherGuests: !calEventRaw.seatsPerTimeSlot
+          ? true
+          : // If there are seats and show attendess is enabled then share attendee information
+          calEventRaw.seatsPerTimeSlot && calEventRaw.seatsShowAttendees
+          ? true
+          : false,
```

Comment:
```suggestion
        guestsCanSeeOtherGuests: !!calEventRaw.seatsPerTimeSlot ? calEventRaw.seatsShowAttendees : true
```
We can use a falsy and just use one ternary.
Also the second condition returns a boolean so ternary it's not required there.
` calEventRaw.seatsPerTimeSlot && calEventRaw.seatsShowAttendees`